### ISSUE TITLE
Enter Key stops working in editor when editing Markdown file

### ISSUE THREAD

**Issue by johnottone** [microsoft/vscode#184662]
Type: <b>Bug</b>

When typing in a markdown file, even on the very fist line, the Enter Key WILL NOT open a new line.  For example:, I create a new markdown file and begin typing...

`# My Title`

after typing the "e" in "Title", I press the Enter key , it can be the one on the primary keyboard or on the the number pad. NEITHER of them drop the cursor to the next line. 

This has been been reported before, at least once and as I am not a collaborator it will not let me coment on that issue post. 
Please see: https://github.com/microsoft/vscode/issues/66611
Also, this has happend before, but was fixed with subsequent VSCode updates.
As for workarounds, you can reload VSCode or you can open the Settings to make the behavior go away, and make the Enter Key immediately start working again. 

There are no steps to reproduce this bug, because it randomly occurs.

I have not installed any new extensions recently, and disabling all extensions did not fix the issue.  

Also I have reviewed my Keyboard Shortcuts and there are only System Sources assigned to every Keybinding with just "Enter" I can post a screenshot if needed.

VSCode Info
Version: 1.78.2 (user setup)
Commit: b3e4e68a0bc097f0ae7907b217c1119af9e03435
Date: 2023-05-10T14:39:26.248Z
Electron: 22.5.2
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.19045
Sandboxed: Yes

VS Code version: Code 1.78.2 (b3e4e68a0bc097f0ae7907b217c1119af9e03435, 2023-05-10T14:39:26.248Z)
OS version: Windows_NT x64 10.0.19045
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10875H CPU @ 2.30GHz (16 x 2304)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|63.64GB (24.66GB free)|
|Process Argv|--crash-reporter-id 1e41e0be-9e57-430a-9774-240d25ac143f|
|Screen Reader|no|
|VM|80%|
</details><details><summary>Extensions (100)</summary>

Extension|Author (truncated)|Version
---|---|---
android-dev-ext|ade|1.3.2
vscode-css-formatter|aes|1.0.2
vscode-sqlite|ale|0.14.1
tsl-problem-matcher|amo|0.6.2
LinkCheckMD|bla|0.3.1
csvtojson-converter|Chu|1.0.2
csharpier-vscode|csh|1.3.6
vscode-database-client2|cwe|6.5.1
vscode-markdownlint|Dav|0.50.0
vscode-eslint|dba|2.4.0
docs-article-templates|doc|1.0.0
docs-authoring-pack|doc|1.0.0
docs-build|doc|0.4.1
docs-images|doc|1.0.0
docs-linting|doc|0.0.13
docs-markdown|doc|1.0.1
docs-metadata|doc|1.0.9
docs-preview|doc|1.0.1
docs-scaffolding|doc|0.0.17
docs-yaml|doc|1.0.0
xml|Dot|2.5.1
vsc-material-theme|Equ|33.8.0
vsc-material-theme-icons|equ|2.8.0
php-intellisense|fel|2.3.14
classroom|git|0.0.3
codespaces|Git|1.14.7
remotehub|Git|0.56.1
vscode-pull-request-github|Git|0.64.0
live-sass|gle|6.0.5
bootstrap5-snippets|Han|1.2.5
vscode-drawio|hed|1.6.6
yaml-plus-json|hil|1.12.1
prettier-sql-vscode|inf|1.6.0
dupchecker|jia|0.1.7
diff-tool|jin|0.0.1
changelog-and-markdown-snippets|jos|1.2.1
click-n-save|llg|1.0.3
rainbow-csv|mec|3.7.0
json|Mee|0.1.2
vscode-azureappservice|ms-|0.24.9
vscode-azureresourcegroups|ms-|0.7.5
vscode-docker|ms-|1.25.1
extension-manifest-editor|ms-|0.1.5
blazorwasm-companion|ms-|1.1.4
csharp|ms-|1.25.9
vscode-dotnet-runtime|ms-|1.6.0
pxt-vscode-web|ms-|1.0.1
data-workspace-vscode|ms-|0.5.0
mssql|ms-|1.19.1
sql-bindings-vscode|ms-|0.4.0
sql-database-projects-vscode|ms-|1.1.1
isort|ms-|2022.8.0
python|ms-|2023.8.0
vscode-pylance|ms-|2023.6.10
remote-containers|ms-|0.292.0
remote-wsl|ms-|0.78.9
azure-account|ms-|0.11.5
azure-repos|ms-|0.32.0
cmake-tools|ms-|1.14.33
cpptools|ms-|1.15.4
cpptools-extension-pack|ms-|1.3.0
hexeditor|ms-|1.9.11
live-server|ms-|0.4.8
powershell|ms-|2023.6.0
remote-repositories|ms-|0.34.0
vsliveshare|ms-|1.0.5864
material-icon-theme|PKi|4.28.0
polacode|pnp|0.3.4
quicktype|qui|12.0.46
filetemplate|Ral|2.0.4
vscode-thunder-client|ran|2.7.5
java|red|1.19.0
vscode-xml|red|0.25.0
vscode-yaml|red|1.13.0
chordpro|ric|0.2.0
LiveServer|rit|5.7.9
synthwave-vscode|Rob|0.1.15
jenkinsfile-support|sec|0.1.0
html5-boilerplate|sid|1.1.1
code-spell-checker|str|2.20.4
Blazor-VSCode-Extensions|Syn|21.2.3
Web-VSCode-Extensions|Syn|21.2.3
ms-teams-vscode-extension|Tea|5.0.0
vscode-markdown-paste-image|tel|1.0.0
remove-blank-lines|tha|1.1.1
cmake|twx|0.0.17
alphabetical-sorter|ue|2.0.1
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
vscode-java-debug|vsc|0.51.0
vscode-java-dependency|vsc|0.23.0
vscode-java-pack|vsc|0.25.11
vscode-java-test|vsc|0.39.0
vscode-maven|vsc|0.41.0
WebTemplateStudio-dev-nightly|WAS|0.7.2117301
faster-new|wen|1.5.2
php-debug|xde|1.32.1
php-pack|xde|1.0.3
template|yon|1.3.0
php-intellisense|zob|1.1.3

(28 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
2e4cg342:30602488
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
2i9eh265:30646982
showlangstatbar:30737416
vsctsb:30748421
pythonms35:30701012
03d35959:30757346
pythonfmttext:30731395
pythoncmv:30756943
fixhidewlkth:30730051
hidesbindicator:30730055
pythongtdpath:30739704
pythonnosmt12:30757131

```

</details>

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.79.0. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by johnottone** [comment]
I finally was able to find the solution to this.  I actually found another [post](https://github.com/microsoft/vscode/issues/100605) talking about 'automaticList ' which was supposed to be removed when I removed "docs-markdown", but wasn't. Its actually in a different extension than reported before. While the older post reports "docs-markdown" as the culprit, it is actually called "Learn Markdown" now. I ended up following the original posters advice and changed its key binding to "CTRL+A+ENTER".
